Ex37b.py

```diff
@@ -0,0 +1,48 @@
+# This script tests all of the data types
+
+#True
+print "This is a true value."
+true_test = True or False
+print true_test
+print "\n"
+
+# False
+print "This is a false value."
+false_test = True and False
+print false_test
+print "\n"
+
+# None
+print "This is the 'none' data type."
+none_test = None
+print none_test
+print "\n"
+
+# string
+print "This is a string value."
+test_string = "Here is the text inside of my string."
+print "\n"
+
+# number
+print "Here is a number."
+x = 007
```

Comment:
Just fyi, since x is not a string. It might remove the trailing 0's because 007 is the same as 7 when it comes to an integer.